activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -196,6 +196,17 @@ def supports_savepoints?
         true
       end
 
+      def supports_insert_returning?
+        true
+      end
+
+      def supports_on_conflict?
```

Comment:
Or would standardising on `unique_by` instead of `conflict` make more sense?

Reply:
👍 I can see the case for `supports_insert_on_conflict?`, @kamipo 

@kaspth, this method isn't about a Rails feature but a Postgres feature. Since version 9.5, Postgres has had the ON CONFLICT syntax. Ever since it's supported that syntax, this adapter has had a way of providing features like "on_duplicate_skip". 

Put another way, in answer to the question, _Does this adapter support insert_on_duplicate_skip?_, we answer _It does if this version of Postgres supports ON CONFLICT_